packages/server/graphql/mutations/helpers/addSeedTasks.ts

```diff
@@ -12,7 +12,7 @@ const NORMAL_TASK_STRING = `This is a task card. They can be created here, in a
 const INTEGRATIONS_TASK_STRING = `Parabol supports integrations for Jira, GitHub, GitLab, Slack and Mattermost. Connect your tools on Settings > Integrations.`
 
 function getSeedTasks(teamId: string) {
-  const integrationURL = makeAppURL(appOrigin, `team/${teamId}/settings/integrations`)
+  const integrationURL = makeAppURL(appOrigin, `team/${teamId}/settings/integrations?from=TeamTask`)
```

Comment:
Let's use UTM, example [here](https://github.com/ParabolInc/parabol/blob/master/packages/server/graphql/mutations/helpers/notifications/notifyMattermost.ts#L88-L92)

```javascript
  const searchParams = {
    utm_source: 'new user seed task card',
    utm_medium: 'product',
    utm_campaign: 'onboarding'
  }
```

Reply:
Yes, updated it!